### DISCUSSION TITLE
v.redd.it support

### DISCUSSION THREAD

**Discussion by gorlilla** [Serene-Arc/bulk-downloader-for-reddit#559] [Ideas]
Was wondering if there are any plans to support v.redd.it hosted submissions? I'm hoping i didn't miss something somewhere, but they don't seem to be picked up when parsing a user's submissions.

**Comment by Serene-Arc** [comment]
They should be, the BDFR has supported them for a long time.

**Reply by c0mplexx92** [reply]
There seems to be an issue with it, it downloads the videos but the name lacks the period before the file extension (MP4 in this case), putting the period before the mp4 ends up only giving the audio instead of both audio and video

**Reply by Serene-Arc** [reply]
Ah, that issue has been resolved recently, though the update hasn't been released yet. I'll do so soon.

**Reply by gorlilla** [reply]
So, I've checked through logs (unique log per user) of user scrapes for 347 user profiles. There are no attempts to use v.redd.it as a source that i can find in those logs (gyfcat, redgifs, imgur/gifv, etc... all seem to work fine). A total of 49,363 unique mp4/jpg were downloaded.

I also parsed the same user profiles using a different "APP" and found links for an additional 6,180 unique mp4 files, all sourced only from v.redd.it.

There were no duplicates found when checking both document sets using hash before merging sets.

The other "APP" is closed source and is not scriptable -- so I neither want to replace BDFR with it nor have to rely on it.

I am using and have only used BDFR 2.4.2.

**Reply by Serene-Arc** [reply]
@gorlilla I'm not sure what you're trying to say here> Are you saying that you don't think the BDFR is downloading them at all or what? We use the YoutubeDLFallback module iirc to download vreddit resources, at least for the past few versions.

**Reply by gorlilla** [reply]
So i apparently am just dumb. I had a separate script for a full profile ingest than what i was looking at (which is for updates). That script had the YoutubeDLFallback module disabled for some reason i forgot. It's working now that I fixed my mistake. Thank you for replying and pointing it out.

**Reply by Serene-Arc** [reply]
Ah! Yes well in the interest of code de-duplication, a VReddit class was removed a while ago since the fallback downloads it perfectly well
